patches/ios-chrome-browser-application_context-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/ios/chrome/browser/application_context/BUILD.gn b/ios/chrome/browser/application_context/BUILD.gn
+index d1733fed5fad2736a13a3478faa24e7ae1cf5417..841883bd81828799ffc89c649e9e4e3b83d5df96 100644
+--- a/ios/chrome/browser/application_context/BUILD.gn
++++ b/ios/chrome/browser/application_context/BUILD.gn
+@@ -9,6 +9,7 @@ source_set("application_context") {
+     "application_context.mm",
+   ]
+   deps = [ "//base" ]
++  public_deps += [ "//brave/ios/browser:chromium_src" ]
```

Comment:
we shouldn't be including brave/ios/browser in ios/chrome/browser/application_context, we need to move things out of ios/chrome/browser to match what upstream has done

Reply:
It looks like all this dep is doing is adding a dep on `//brave/common` and the only place I see it's needed in for iOS there is in the `brave/chromium_src/ios/chrome/browser/net/ios_chrome_network_delegate.cc` override. So maybe we can just add a `sources.gni` to `brave/chromium_src/ios/chrome/browser/net` and patch the corresponding upstream BUILD.gn?

Reply:
we don't want sources.gni unless there's a circular dep so it sounds like we just need to move `//brave/ios/browser:chromium_src` -> `//brave/ios/browser/application_context:chromium_src`

Reply:
wait, why are we adding this to ios/chrome/browser/application then?